V2.2

MingmingZHAO opened the pull request.

seinlin committed:
Bug 1119671 - Update Myanmar keyboard layouts and add imes for it.

KevinGrandon committed:
Bug 1128275 - Disable intermittent test in calendar/test/unit/day_observer_test.js r=kgrandon

janjongboom committed:
Merge pull request #27747 from comoyo/remove_demo_keyboard

Bug 1126012 - Remove dev_apps/demo-keyboard. r=djf
Conflicts:
	dev_apps/demo-keyboard/test/unit/worker_test.js
	tools/pre-commit

phoenix1moz committed:
Bug 1138278 - [v2.2][Clock] After running test_clock_create_new_alarm.py, it should set the alarm time to 1 min more than the current time

KevinGrandon committed:
Bug 1130552 - Disable broken test, TEST-UNEXPECTED-FAIL | null | [privacy-panel-test/unit/ala/blur_slider_test.js] ALA BlurSlider "before all" hook

mikehenrty committed:
Bug 1126613 - Ensure utility tray always opens to left of SHB no matter doc dir

KevinGrandon committed:
Bug 1097633 - [Search] Hide theme apps by default

alivedise committed:
Merge pull request #28885 from alivedise/bugzilla/1131234/simlock-after-lock-keyboard

Bug 1131234 - Add simlock events back, r=alberto

KevinGrandon committed:
Bug 1146078 - Include fb_resolver.js to avoid races with imgLoader being undefined

ian-liu committed:
Bug 1145031 - fix duplicated option in select option of server certificate, r=ej

Commit:
Bug 1144655 - Closing search view when entering in the task manager

daleharvey committed:
Bug 1146230 - Add ignoreFront to getScreenshot to get browser window. r=alive

shinglyu committed:
Bug 1149024 - wait_for_notification_toaster_displayed needs 'message' as its argument

stasm committed:
Bug 994357 - Use DOM overlays for safely setting node content. r=gandalf

millermedeiros committed:
Bug 1149087 - [Calendar] center time on Day View sidebar.

KevinGrandon committed:
Bug 1150495 - [GaiaGrid] Do not overwrite icons when we have a cached icon

Commit:
Bug 1147107 - Update mozilla-download to fetch builds from taskcluster index instead of ftp r=lightsofapollo

jrburke committed:
Bug 1149070 - [Text Selection] Right caret disappeared in email select all

albertopq committed:
Bug 1150424 - Preventing default on top-panel click and mouse events

mancas committed:
Bug 1154439 - [STK] TypeError: options.confirmMessage is undefined at: app://system.gaiamobile.org/gaia_build_defer_index.js. r=timdream

albertopq committed:
Bug 1150025 - Dismissing SIM dialog when opening the rocketbar

Commit:
Bug 1013594 - [Video] When selecting a video in the Video App, the video will flicker blue

weilonge committed:
Bug 1154649 - Switch transition property when entering/leaving edit-mode.

gabrielesvelto committed:
Bug 1155582 - Sync the phone and email details fragments used when displaying a contact in the call log information pane r=drs,francisco

Commit:
Bug 1151496 - [RTL][Systems] Parentheses in song title are not displayed correctly on ringtone-related screens

sfoster committed:
Bug 1155568 - Allow normal line-height for wifi entries. r=fcampo, a=jocheng

msliu committed:
Bug 1156141 - Make switch from boot animation to first system app screen seamlessly. r=alive

cctuan committed:
Bug 1155873 - prevent home and holdhome event while shrinking ui is active, r=alive

Commit:
Bug 1151775 - [Flame][Video]Share a video to mail/Messages and then back to Video, the video can't be played

mikehenrty committed:
Bug 1156701 - Update download icons to be dark against white bg

etiennesegonzac committed:
Bug 1155785 - App Windows should not overflow. r=alive

albertopq committed:
Bug 1133029 - Resizing dialogs on hierarchy change

sfoster committed:
Bug 1157610 - Bidi entry sheet titles

albertopq committed:
Bug 1157157 - Using 2 lines on FTU FB import messages. r=sfoster, a=jocheng

gmarty committed:
Bug 1157630 - [Flame][E.me Integration] Status bar will not disappear on screenshot preview page

Commit:
Bug 1156161 - Add manifestHash and packageHash fields in webapps.json

daleharvey committed:
Bug 1157128 - Fix display of auth inputs. r=kgrandon

Commit:
Bug 1158396 - fix a race condition in the image transition code

sfoster committed:
Bug 1157173 - Rejigger change button styles. r=mhenretty. a=bbajaj

sean2449 committed:
Bug 1158715 - XSS/HTML injection in TV Smart System app

gasolin committed:
Bug 1154209 - [l10n][Camera]Arabic:The checkboxes will be overlapped with the text in the Website Permission view, r=alive

etiennesegonzac committed:
Bug 1157615 - 2 fingers taps should never trigger edge gestures. r=alive

mcav committed:
Bug 1152683 - [email] reply-to header ignored. r=asuth

mikehenrty committed:
Bug 1157159 - Adjust button padding to account for gecko patch in bug 1010675

Commit:
Bug 1159092 - get ellipsis on the correct side for lockscreen and notification screen playback controls.

fix the new tests

sean2449 committed:
Bug 1159136 - [Stingray][Smart-System] Replace innerHTML with textContent in app_modal_dialog

cctuan committed:
bug 1158379 - hideFrame when the image of screenshot is ready to show, r=alive

Commit:
Bug 1157167 - [Flame][Contacts]ICE contacts icon will disappear in contacts list page after user sets "Order by last name". r=ferjm

rvandermeulen committed:
Revert "Bug 1156161 - Add manifestHash and packageHash fields in webapps.json"

This reverts commit bf6670f17f25722b2c3c4403beb9e5017cf7a67c.

rvandermeulen committed:
Bug 1157630 - Add back the test change that was lost during rebasing.

hfiguiere committed:
Bug 1158043 - Fix ellipsis in the song title.

Commit:
Bug 1158093 - [Contacts]Russian:The birthday field does not fit screen width. r=ferjm

sfoster committed:
Bug 1158740 - reduce padding in dialog buttons. r=kgrandon, a=jocheng

Commit:
Bug 1158727 - Fix Emergency Call keypad deletion when text cursor is placed. r=thills

marshall committed:
Bug 1134998 - Ignore private window events when collecting app usage. r=djf

sean2449 committed:
Bug 1159137 - [Stingray][Smart-System] Replace innerHTML with textContent in ValuePicker to prevent XSS injection

Commit:
Bug 1156161 - Add manifestHash and packageHash fields in webapps.json

Commit:
Merge pull request #29812 from RickyChien/add-manifestHash-and-packageHash-v2.2

Bug 1156161 - Add manifestHash and packageHash fields in webapps.json r=gduan

EragonJ committed:
Bug 1155600 - add missing happy field & fix test r=Arthur

KevinGrandon committed:
Bug 1159137 - Cherry pick bug 1159137, use textContent for pickerUnit

KevinGrandon committed:
Bug 1159136 - Cherry pick bug 1159136, use textContent for app modal dialog

KevinGrandon committed:
Bug 1159136 - Cherry pick bug 1159136, use textContent for app modal dialog

jrburke committed:
Bug 1159982 - [RTL][Email] Long email address is not correctly displayed on email address dialog

rvandermeulen committed:
Merge pull request #29800 from jimporter/ringtone-rtl-2.2

Bug 1151496 - [RTL][Systems] Parentheses in song title are not displayed correctly on ringtone-related screens

rvandermeulen committed:
Merge pull request #29773 from dmarcos/bug1157169

Bug 1157169 - [Camera] The Self-timer overlaps with the Off

gabrielesvelto committed:
Bug 1135284 - Do not show the MMI result overlay in response to call control supplementary service commands r=thills

mozilla-autolander-deprecated committed:
Bug 1158447 - fix SMTP dot stuffing. v2.2-specific, r=mcav

mcav committed:
Bug 1145473 - [email/backend] MailAPI's bridge-processing logic may break (v2.2) r=asuth

sfoster committed:
Bug 1159644 - Reduce padding on buttons to account for fixed overflow/truncation behavior. r=kgrandon, a=bbajaj

sfoster committed:
Bug 1160096 - Fix confirm button padding. r=kgrandon, a=jocheng

albertopq committed:
Bug 1160917 - Recalculate fullscreen on activityopened event

gabrielesvelto committed:
Bug 1160337 - Properly end a single, held conference call r=drs

aosmond committed:
Bug 1157163 - Fix how camera may be reopened too early when using the preview gallery. r=justindarc

mcav committed:
Merge pull request #29831 from mcav/reportError-v2.2

Bug 1145473 - [email/backend] MailAPI's bridge-processing logic may break. r=asuth a=2.2+

sfoster committed:
Bug 1157174 - Reduce end-padding for menu buttons. r=kgrandon, a=bbajaj

millermedeiros committed:
Bug 1160363 - [Calendar][Week Bar] Tapping on the bar at the top with the initials of the days of the week will bring the user to the 1st day of the month

delapuente committed:
Bug 1160348 - [Usage][cost control][text selection] A result of NaN becomes entered into the Data Usage Limit text box and can not be erased when selecting the paste option

mikehenrty committed:
Bug 1160667 - Cancel install when opening lockscreen

crh0716 committed:
Bug 1158094 - Remove ellipsis from the list items

albertopq committed:
Bug 1160505 - Fixing ui tests with long alerts

albertopq committed:
Bug 1109580 - Fix intermitent icc unit tests

Cwiiis committed:
Bug 1155091 - Fix music controls causing utility tray flicker. r=kgrandon

sfoster committed:
Bug 1155459 - Add bdi in popup headers

steveck-chung committed:
Bug 1160232 - [Message] Set recipient pill content unselectable, r=julienw

etiennesegonzac committed:
Bug 1160191 - Dialogs inside an appWindow element don't need to take the SHB into account. r=kgrandon

gabrielesvelto committed:
Bug 1159958 - Do not check for the number of existing calls before trying to dial r=thills

mikehenrty committed:
Merge pull request #29835 from mikehenrty/gmarty-Bug-1159885-Status-bar-is-not-visible-on-multiple-attention-windows

Gmarty bug 1159885 status bar is not visible on multiple attention windows
Conflicts:
	apps/system/test/unit/statusbar_test.js

sfoster committed:
Bug 1147315 - No resize for 180 orientation changes. r=etienne. a=jocheng

lightsofapollo committed:
NO BUG - Switch to new provisioner r=me a=npotb

lightsofapollo committed:
Merge pull request #29929 from lightsofapollo/new-tc-provisioner

NO BUG - Switch to new provisioner r=me a=npotb

weilonge committed:
Bug 1162224 - Fix the way to recognize Browser app terminated.

crh0716 committed:
Bug 1160649 - Force to use wrapping in the feedback textarea

crh0716 committed:
Bug 1162276 - Display the current timezone information when auto timezone is enabled

When auto timezone is enabled, we are not able to tell the region and city from the timzone information. In this case we show the information itself and hide the selectors for choosing region and city.

crh0716 committed:
Bug 1162276 - Display the current timezone information when auto timezone is enabled

When auto timezone is enabled, we are not able to tell the region and city from the timzone information. In this case we show the information itself and hide the selectors for choosing region and city.

KevinGrandon committed:
Bug 1161452 - Use action attribute for icc confirm header

rvandermeulen committed:
Merge pull request #29940 from albertopq/1144655-rocketbar-app-uplift-v2.2

Bug 1144655 - Closing search view when entering in the task manager

rvandermeulen committed:
Merge pull request #29814 from EragonJ/v2.2-bug-1155600

Bug 1155600 - add missing happy field & fix test r=Arthur

Commit:
Bug 1161975 - [Contacts]If we import contact with birthday in January from outlook, the birthday field will be missing. r=ferjm

lightsofapollo committed:
Bug 1162630 - Update make b2g to download gecko v2.2

lightsofapollo committed:
Merge pull request #29951 from lightsofapollo/actually-use-2.2

Bug 1162630 - Update make b2g to download gecko v2.2

sfoster committed:
Bug 1162440 - Adjust confirm button padding. r=dflanagan, a=bbajaj

mikehenrty committed:
Bug 1162768 - Disable test_music_change_rating.py on v2.2

mikehenrty committed:
Merge pull request #29956 from mikehenrty/bug-1162768-uplift-fixes

Bug 1162768 - Uplifting all changes to test_music_change_rating.py

mikehenrty committed:
Merge pull request #29957 from mikehenrty/bug-1126012-remove-demo-keyboard-2.2

Merge pull request #27747 from comoyo/remove_demo_keyboard, a=test-only

MDTsai committed:
Bug 1161140 - Add tosterClicked to show download dialog

Also add unit test for toaster clicked, verify utility tray is
hidden and download dialog is shown

mozilla-autolander-deprecated committed:
Bug 1157163 - merge pull request #29864 from aosmond:bug1157163-alt-v2.2 to mozilla-b2g:v2.2

cctuan committed:
Bug 1158388 - [NFC] Swiping up causes jerky, stuttered animation in the Shrinking UI. r=greg

etiennesegonzac committed:
Bug 1162656 - Follow up to bug 1160191. Nested dialog should not take SHB into account in landscape too. r=albertopq

Commit:
Bug 1163187 - remove scroll_detector.js and the -moz-crisp-edges scrolling code

albertopq committed:
Merge pull request #29986 from albertopq/1157128-uplift-v2.2

Bug 1157128 - Fix display of auth inputs. r=kgrandon

albertopq committed:
Merge pull request #29998 from albertopq/1150025-uplift-v2.2

Bug 1150025 - Dismissing SIM dialog when opening the rocketbar

mwargers committed:
Bug 1161441 - [Gaia::UI Tests] test_gallery_flick.py failed, v2.2 version

hfiguiere committed:
Bug 1163679 - Remove the -moz-crisp-edge anti optimization

Commit:
Bug 1163215 - switch MediaFrame back to using <img> instead of background-image

update comments to match code changes

re-use a single image element and keep it hidden before the image has loaded

fix the python tests

remove the test for the corrupt jpeg image

tedders1 committed:
Bug 1154438 - Wrap subsitutions in bidi isolate characters. r=stas

Commit:
Bug 1163784 - (v2.2) Highlight Bluetooth annuciator after connecting to remote device

Commit:
Merge pull request #30016 from bentian/bug-1163784-22

Bug 1163784 - (v2.2) Highlight Bluetooth annunciator after connecting to remote device, r=iliu, a=kkuo

azasypkin committed:
Bug 1156464 - [Window Management] Accessing a message thread via notification after killing Messages app creates adverse effects on future notifications while in app. r=schung

rvandermeulen committed:
Merge pull request #29871 from asutherland/email-dot-stuffing-v22

Bug 1158447 - fix SMTP dot stuffing. v2.2-specific, r=mcav

rvandermeulen committed:
Merge pull request #29895 from marshall/bug1134998_aumPrivateWindows_v2.2

Bug 1134998 - Ignore private window events when collecting app usage

cctuan committed:
Bug 1162384 - [NFC] Cannot share image which is opened from notification, r=alive

EragonJ committed:
Bug 1161785 - Fix anchor CSS selector in lists.css

gerrycfchang committed:
Bug 1136791 - test_marketplace_packaged_app times out after opening Marketplace, unable to search

gerrycfchang committed:
Bug 1151902 - test_dialer_find_contact.py: "IndexError: list index out of range"

albertopq committed:
Merge pull request #30019 from arcturus/bug-1146078

Bug 1146078 - Include fb_resolver.js to avoid races with imgLoader being undefined a=testonly

albertopq committed:
Merge pull request #30042 from albertopq/1160917-statusbar-rocketbar-uplift-v2.2

Bug 1160917 - Recalculate fullscreen on activityopened event

mikehenrty committed:
Merge pull request #30044 from mikehenrty/bug-1159885-uplift

Merge pull request #29835 from mikehenrty/gmarty-Bug-1159885-Status-b…

etiennesegonzac committed:
Bug 1158928 - Removing old layer trick and making sure cards always transition their transform r=sfoster

mozilla-autolander-deprecated committed:
Bug 1156464 - merge pull request #29922 from azasypkin:v2.2-bug-1156464-notification-hash to mozilla-b2g:v2.2

sfoster committed:
Bug 1160080 - Add isolation characters around URLs in net error msgs

jrburke committed:
Bug 1163741 - [Email] LTR subject line has end punctuation placed to left of string when language is set to RTL

mwargers committed:
Merge pull request #29241 from shinglyu/bug1149024

Bug 1149024 -  [v2.2] wait_for_notification_toaster_displayed needs 'message' as its argument

jrburke committed:
Bug 1165205 - [RTL][Email]The title in Google login page is truncated at left side

rvandermeulen committed:
Merge pull request #30037 from gerrycfchang/bug1136791

Bug 1136791 - test_marketplace_packaged_app times out after opening M…

HermesCheng committed:
Bug 1149886 - using <bdi> to replace <span> at Music app results in unable to locate <span.list-song-index> error. r=jlorenzo

Commit:
Bug1115180 - Get test_cost_control_data_alert_mobile.py re-enable

mwargers committed:
Merge pull request #30096 from HermesCheng/v2.2

Bug 1149886 - using <bdi> to replace <span> at Music app results in unable to locate <span.list-song-index> error. r=jlorenzo

mwargers committed:
Merge pull request #30100 from edwardchen40/bug1115180

Bug 1115180 - Get test_cost_control_data_alert_mobile.py re-enabled

Cwiiis committed:
Bug 1147316 - Fix cut-off add-to-collection icon. r=kgrandon

mozilla-autolander-deprecated committed:
Bug 1162732 - merge pull request #30065 from gabrielesvelto:bug-1162732-speaker-and-mute-button-status to mozilla-b2g:master

(cherry picked from commit 23b3a165ce3bc4fc5e0ef63b8ad45c63ce636723)

Conflicts:
	apps/callscreen/js/calls_handler.js

gmarty committed:
Bug 1102183 - Signal strength and Data signal (3G/H/E) is not changing while checking in after opening any app

rvandermeulen committed:
Bug 1163215 - Follow-up Linter fix. a=bustage

EragonJ committed:
Bug 942365 - fix done button problem in different scenario, r=Arthur

EragonJ committed:
Merge pull request #30115 from EragonJ/v2.2-bug-942365

Bug 942365 - fix done button problem in different scenario, r=Arthur

rvandermeulen committed:
Revert "Bug 1162276 - Display the current timezone information when auto timezone is enabled"

This reverts commit 13b168747ba9666c03b7203bb9a14a30c623a380.

gabrielesvelto committed:
Merge pull request #30127 from gabrielesvelto/bug-1162732-v2.2-uplift

Bug 1162732 - v2.2 uplift

sfoster committed:
Bug 1161456 - Handle bidi wifi ids. r=fcampo, a=jocheng

millermedeiros committed:
Bug 1165829 - [RTL][Calendar] The heading of Google login page is truncated at wrong side in Calendar

alivedise committed:
Bug 1166591 - Drop browser element ref when app is suspended

albertopq committed:
Bug 1163169 - Blocking home events on Logo handler

mwargers committed:
Merge pull request #30009 from mwargers/1161441_v2.2

Bug 1161441 - [Gaia::UI Tests] test_gallery_flick.py failed, v2.2 version

mwargers committed:
Merge pull request #30040 from gerrycfchang/bug1151902

Bug 1151902 - test_dialer_find_contact.py: "IndexError: list index out of range"

Commit:
Bug 1163789 - work around gecko CSS animation bugs for v2.2

address review comments, fix broken test

mantaroh committed:
Bug 1167050 - Remove container hidden code when choose datetime/datetime-local picker. r=alive

rvandermeulen committed:
Merge pull request #30162 from davidflanagan/bug1163789-v2.2

Bug 1163789 - work around gecko CSS animation bugs for v2.2

alivedise committed:
Bug 1166595 - Blur the activity opener before opening activity window

Commit:
Bug 1164421 - autocorrect im to I'm not in

address review comments

(cherry picked from commit 79e987f19487990de3a53401be85866e464e6f49)

fix gjslint error

crh0716 committed:
Merge pull request #30138 from crh0716/1162276_v2.2

Bug 1162276 - Display the current timezone information when auto time zone is enabled r=eragonj

ian-liu committed:
Bug 1166636 - remove device paired from found device list of background app while it is paired via the foreground Bluetooth/Settings app

Commit:
Bug 1167336 - use <img> for Video app thumbnails instead of background-image

update failing unit test

(cherry picked from commit d2a2b30b61e51efe27a521d2a9b815be4e9da49f)

updated to work with the old templating system in 2.2

rvandermeulen committed:
Merge pull request #30200 from davidflanagan/bug1164421-v2.2

Bug 1164421 - autocorrect im to I'm not in

albertopq committed:
Bug 1163524 - Closing search when opening an activity

mozilla-autolander-deprecated committed:
Bug 1167336 - merge pull request #30210 from davidflanagan:bug1167336-v2.2 to mozilla-b2g:v2.2

alivedise committed:
Merge pull request #30233 from alivedise/bugzilla/1166591-v2.2real

Bug 1166591 - Drop browser element ref when app is suspended

cctuan committed:
Bug 1163957 - [NFC]Can't share screenshot from notification preview page.

crh0716 committed:
Revert "Bug 1158094 - Remove ellipsis from the list items"

This reverts commit 1ed6b0fced5f0440a7c939622a255498b7f7448c.

albertopq committed:
Bug 1165849 - Resizing app_window properly on rocketbar activities

Commit:
Bug 1165824 - [RTL][Contacts]The name and address are truncated at wrong side at select Gmail contacts view. r=ferjm

wilsonpage committed:
Bug 1161364 - Updated gaia-header to v0.8.0-1

rvandermeulen committed:
Merge pull request #30255 from gmarty/Bug-1102183-Signal-strength-and-Data-signal-is-not-changing-while-checking-in-after-opening-any-app-v2.2

Bug 1102183 - Signal strength and Data signal (3G/H/E) is not changin…

rvandermeulen committed:
Revert "Bug 1159136 - Cherry pick bug 1159136, use textContent for app modal dialog"

This reverts commit 366f2204e918b76c0d4a4990473d1e6ec68c9063.

KevinGrandon committed:
Merge pull request #30272 from KevinGrandon/bug_1160069_uplift_2_2

Bug 1159136 - Cherry pick bug 1159136, use textContent for app modal dialog

mozilla-autolander-deprecated committed:
Bug 1114864 - merge pull request #29916 from RudyLu:keyboard/Bug1114864-igbo_layout to mozilla-b2g:master

(cherry picked from commit be82b6edc1221f041f282b68570bbe3975b5ffec)

Conflicts:
	apps/keyboard/test/build/integration/keyboard_test.js
	apps/keyboard/test/build/integration/resources/all-layout-make-layouts.json
	apps/keyboard/test/build/integration/resources/default-make-layouts.json
	apps/keyboard/test/build/integration/resources/no-preload-dict-required-make-layouts.json

crh0716 committed:
Bug 1158094 - Allow multi-line for menu-items

gabrielesvelto committed:
Bug 1126260 - Do not free the callscreen when memory is low if it's in active use

rvandermeulen committed:
Merge pull request #30166 from cctuan/1163957-v22

Bug 1163957 - [NFC]Can't share screenshot from notification preview page.,r=djf

dominickuo committed:
Bug 1168549 - [Music] use <img> to replace background-image

dominickuo committed:
Bug 1168549 - [Music] use <img> to replace background-image

rvandermeulen committed:
Merge pull request #30263 from wilsonpage/1161364-v2.2

Bug 1161364 - Updated gaia-header to v0.8.0-1

atiqueahmedziad committed:
Bug 1127559 - [Dialer] End call button gets wider when putting a call on hold

Now the End call button doesn't get wider after putting a call on hold

mwargers committed:
Merge pull request #30192 from Lance-k/1138278

Bug 1138278 - [v2.2][Clock] After running test_clock_create_new_alarm.py, it should set the alarm time to 1 min more than the current time

ferjm committed:
Bug 1165248 - [FxA] The long email address will be cut off in 'Confirm Your Account' view. r=arthurcc

Commit:
Bug 1039893 - Enable screen before setScreenBrightness. r=timdream, a=jocheng

rvandermeulen committed:
Bug 1039893 - Add back inadvertently-removed comment.

cynthiatang committed:
Bug 1166330 - test_sms_with_picture_attached.py TimeoutException

mwargers committed:
Merge pull request #30358 from cynthiatang/v2.2

Bug 1166330 - test_sms_with_picture_attached.py TimeoutException

sfoster committed:
Bug 1170102 - Right-align labels in RTL wifi list

cynthiatang committed:
Bug 1168284 - test_marketplace_packaged_app.py TimeoutException

cynthiatang committed:
Bug 1166631 - test_music_artist_mp3.py TimeoutException

mwargers committed:
Merge pull request #30384 from cynthiatang/bug1166631

Bug 1166631 - test_music_artist_mp3.py TimeoutException

mwargers committed:
Merge pull request #30382 from cynthiatang/bug1168284

Bug 1168284 - test_marketplace_packaged_app.py TimeoutException

sfoster committed:
Merge pull request #30373 from sfoster/wifi-rtl-labels-bug-1170102

Bug 1170102 - Wifi RTL labels. r=tedders, a=jocheng

rvandermeulen committed:
Merge pull request #30309 from dominickuo/v2.2-bug-1168549

Bug 1168549 - [Music] use <img> to replace background-image (v2.2)

dominickuo committed:
Revert "Bug 1168549 - [Music] use <img> to replace background-image"

This reverts commit cdd48c73ad78aa5d6785d91bbe100064d39fa168.

RudyLu committed:
Merge pull request #30274 from RudyLu/ltedone-Bug_1137968-Create_Ukrainian_keyboard

Bug 1137968 - [B2G] Create Ukrainian keyboard.
r=rudylu
(cherry picked from commit 45afe9b99481d768d90a74a5322389f5873d844f)

Conflicts:
	apps/keyboard/test/build/integration/keyboard_test.js
	apps/keyboard/test/build/integration/resources/all-layout-make-layouts.json
	apps/keyboard/test/build/integration/resources/default-make-layouts.json
	apps/keyboard/test/build/integration/resources/no-preload-dict-required-make-layouts.json

dominickuo committed:
Merge pull request #30400 from dominickuo/v2.2-bug-1168549-new

Bug 1168549 - [Music] use <img> to replace background-image, r=jimporter

punamdahiya committed:
Bug 1157147 - [Gallery][RTL] File name in open view should not be mirrored

mwargers committed:
Bug 1171825 - Cannot import name TreeherderRequest

dominickuo committed:
Bug 1172403 - [Music] Disable the fade-in animation for album art

mwargers committed:
Merge pull request #30472 from mwargers/1171825

Bug 1171825 - Cannot import name TreeherderRequest - v2.2 version

Commit:
[Bug 1171202] Music app's scanning process can stall from invalid files

cynthiatang committed:
Bug 1170941 - test_sms_with_picture_attached.py TimeoutException

mwargers committed:
Merge pull request #30494 from cynthiatang/bug1170941_2

Bug 1170941 - test_sms_with_picture_attached.py TimeoutException

rvandermeulen committed:
Merge pull request #30493 from jimporter/music-invalid-files-2.2

[Bug 1171202] Music app's scanning process can stall from invalid files

dominickuo committed:
Merge pull request #30475 from dominickuo/v2.2-bug-1172403

Bug 1172403 - [Music] Disable the fade-in animation for album art, r=jimporter

rvandermeulen committed:
Merge pull request #30611 from RudyLu/keyboard/Bug1114864-igbo_layout_v2.2

Bug 1114864 - merge pull request #29916 from

RudyLu committed:
Merge pull request #30256 from RudyLu/keyboard/Bug1153136-Add_Frisian_layout

Bug 1160883 - Add Frisian (fy or fy-NL) wordlist/dictionary.
r=timdream.
(cherry picked from commit b6c5fda821dce99babf8b9d5799a95eb4aeca64a)

Conflicts:
	apps/keyboard/test/build/integration/resources/all-layout-make-layouts.json
	apps/keyboard/test/build/integration/resources/default-make-layouts.json
	apps/keyboard/test/build/integration/resources/no-preload-dict-required-make-layouts.json

Lance-k committed:
Fix Bug 1174268

mwargers committed:
Merge pull request #30667 from Lance-k/1174268

Bug 1174268

shamenchens committed:
Bug 1168256 - Fix search during scanning music in tile/list view

gmarty committed:
Bug 1085651 - Incorrect time (clock) and status showed in the statusbar

rvandermeulen committed:
Merge pull request #30718 from gmarty/Bug-1085651-Incorrect-time-and-status-showed-in-the-statusbar-v2.2

Bug 1085651 - Incorrect time (clock) and status showed in the statusbar

rvandermeulen committed:
Merge pull request #30706 from shamenchens/Bug1168256-MusicSearchInTile-v22

Bug 1168256 - Fix search during scanning music in tile/list view

rvandermeulen committed:
Merge pull request #30723 from RudyLu/keyboard/Bug1160883-Frisian_layout_v2.2

Merge pull request #30256 from RudyLu/keyboard/Bug1153136-Add_Frisian_layout

rvandermeulen committed:
Merge pull request #30722 from RudyLu/keyboard/Bug1137968-Ukrainian_keyboard_v2.2

Merge pull request #30274 from

KevinGrandon committed:
Bug 1179085 - Disable perma-failing test, month_day_test.js | Views.MonthDay #create future-other-month

dominickuo committed:
Bug 1178645 - [Music] Add the missing image_utils.js

rvandermeulen committed:
Merge pull request #30799 from albertopq/1163524-uplift

Bug 1163524 - Closing search when opening an activity

ngokevin committed:
Bug 1172197 - update marketplace packages for openmobile webactivity

fabricedesre committed:
Merge pull request #30809 from ngokevin/v2.2

Bug 1172197 - update marketplace packages for openmobile webactivity (v2.2)

rvandermeulen committed:
Merge pull request #30852 from Seinlin/bug-1119671-v2.2

Bug 1119671 - Update Myanmar keyboard layouts and add imes for it.

TheoChevalier committed:
Bug 1183379 - Add Amharic (am), Upper Sorbian (hsb), Lower Sorbian (dsb), Miahuatlán Zapotec (zam) locales to Gaia on master and 2.2 branches

Commit:
Bug 1182035 - Fixed so when only there are app updates and update.2g.enabled is false and device is in 2G, they are not blocked

mwargers committed:
Bug 1174892 - Gaiatest needs to be updated for treeherder client changes, v2.2 version

mwargers committed:
Merge pull request #30986 from mwargers/1174892_v2.2

Bug 1174892 - Gaiatest needs to be updated for treeherder client changes, v2.2 version

rvandermeulen committed:
Merge pull request #30971 from mcjimenez/bug1182035_v2_2

Bug 1182035 - Fixed app updates error in 2G

snowmantw committed:
Bug 1188716 - fix: add a defer to block user using the slider

rvandermeulen committed:
Merge pull request #31148 from snowmantw/bug1188716

Bug 1188716 - fix: add a defer to block user using the slider

snowmantw committed:
Bug 1190079 - fix: remove defaults; and put all setting reads in a Promise all and delay unlocker initialization

stasm committed:
Bug 1190038 - Sanitize all HTML elements. r=gandalf

Commit:
Bug 1190139 - Remove .innerHTML from wantToReceiveFile entity

rvandermeulen committed:
Revert "Bug 1147107 - Update mozilla-download to fetch builds from taskcluster index instead of ftp r=lightsofapollo"

This reverts commit cd6b333b004c155e9dc611ae27e6c5ea8a4bb3ff.

autolander commented:
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

Commit:
Fixing lockscreen delay bypass by rewriting core logic

This is fixing bug 1181571 where lockscreen delay on repeated pass code
failure could be trivially bypassed by pressing the home button. The
faulty behavior was caused by a limitation of the previous architecture
which was only making the delay occurr *after* a wrong pass code had
been entered. Consequently, whenever the input pad would be restarted,
like after closing it with the home button or when a screen timeout
occurred, the pending delay would not persist and pass code entry
would be accepted right away.

The fix made it necessary to rewrite parts of lockscreen's messaging
and state handling. Previous changes in architecture caused pass
code failure counting and delay timer handling to be maintained in two
places: in LockScreen (for pass code checking) and in LockScreenInputPad
(for displaying error state to the user). Logic error even caused the
two states to get out of sync under certain conditions.

Now state and timeout related to pass code checking are maintained
exclusively by LockScreen, which signals LockScreenInputManager with
three distinct messages of failure (on wrong pass code), reset (after
error delay has passed), and success (on correct pass code).

LockScreenInputPad now is always aware of a passcode error delay ongoing
in LockScreen, every UI update is rendered according to this state, and
key input is ignored when necessary.

Commit:
Fix indentation

Commit:
Make FindMyDevice enforce immediate lock

This addresses bug 1186100 by implementing a hotfix in FindMyDevice.
See tracking bug 1202484. Test was amended to check for the fixed
behavior.

Commit:
Fix passcode override bug 1201530

This is a trivial hotfix, removing the check for an existing pass code,
allowing FMD to override. Also adds the required test.

BavarianTomcat committed:
Merge pull request #31248 from snowmantw/bug1190079-2.2

Bug 1190079 - fix: remove defaults; and put all setting reads in a Promise all and delay unlocker initialization. r=timdream, a=mpotharaju

autolander commented:
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

BavarianTomcat committed:
Merge pull request #31609 from cr/pctofix-v2.2

Bug 1181571 - Fixing lockscreen delay issue, v2.2 backport. r=gweng, a=mpotharaju

autolander commented:
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

BavarianTomcat committed:
Merge pull request #31729 from cr/immediatelockfix-v2.2

Bug 1202484 - Make FindMyDevice enforce immediate lock. r=gerard-majax a=mpotharaju

autolander commented:
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

autolander commented:
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

Commit:
Bug 1204859 - fixing test

Commit:
Bug 1195872 - Workaround mozilla-download breakage after FTP shutdown r=aus, a=test-only

Commit:
NO BUG -- Fix linter errors. a=test-only

nullaus committed:
Merge pull request #31855 from nullaus/v2.2

Bug 1195872 - Workaround mozilla-download breakage after FTP shutdown…

autolander commented:
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

BavarianTomcat committed:
Merge pull request #31851 from cr/bug1204859

Bug 1204859 - fixing test. a=sheriffduty

autolander commented:
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

TheoChevalier committed:
Bug 1189532 - Add Bambara (bm) and Malagasy (mg) locales to master and 2.2 builds, r=stas a=josh

BavarianTomcat committed:
Merge pull request #31889 from TheoChevalier/add-mg-bm

Bug 1189532 - Add Bambara (bm) and Malagasy (mg) locales to master an…. r=jocheng

autolander commented:
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

BavarianTomcat committed:
Merge pull request #31731 from cr/fmdpasscodefix-v2.2

Bug 1201530 - Fix passcode override bug in FMD (v2.2 backport). r=gerald_max, a=mpotharaju

autolander commented:
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

BavarianTomcat committed:
Revert "Fix passcode override bug 1201530" for breaking GU Tests

This reverts commit 672411d07b85dbeea1b0a496293ffe71f5481bf8.

autolander commented:
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

Commit:
Bug 1201530 - Fix FMD passcode ovveride

This is a new pull request on top of the backed-out patch that fixes
the failing test.

BavarianTomcat committed:
Merge pull request #32267 from cr/fmdpasscodefix-v2.2

Bug 1201530 - Fixing FMD passcode override. r=gerard-majax

autolander commented:
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].
